syntax = "proto3";

package api;

service PrimeStream {
  rpc GetPrimes(PrimeCount) returns (stream PrimeNumber) {}
}

message PrimeCount {
  int64 number = 1;
}

message PrimeNumber {
  int64 count = 1;
  int64 value = 2;
}
